content/en/containers/guide/compose-and-the-datadog-agent.md

```diff
@@ -67,6 +67,70 @@ FROM gcr.io/datadoghq/agent:latest
 ADD conf.d/redisdb.yaml /etc/datadog-agent/conf.d/redisdb.yaml
 ```
 
+### APM Trace Collection
+
+Building on the Redis example above, we can also use Compose to configure the Datadog agent to collect application traces. This `docker-compose.yml` is pulled from [this Docker compose example you can find on our GitHub page][4].
+
+```yaml
+version: "4"
+services:
+  web:
+    build: web
+    command: ddtrace-run python app.py
+    ports:
+     - "5000:5000"
+    volumes:
+     - ./web:/code # modified here to take into account the new app path
+    links:
+     - redis
+    environment:
+     - DATADOG_HOST=datadog # used by the web app to initialize the Datadog library
+     - DD_AGENT_HOST=dd-agent # points to dd-agent to send traces
+  redis:
+    image: redis
+  # agent section
+  datadog:
+    container_name: dd-agent
+    build: datadog
+    links:
+     - redis # ensures that redis is a host that the container can find
+     - web # ensures that the web app can send metrics
+    environment:
+     - DD_API_KEY=__your_datadog_api_key_here__
+     - DD_DOGSTATSD_NON_LOCAL_TRAFFIC=true # enables agent to receive custom metrics from other containers
+     - DD_APM_ENABLED=true # enables tracing
+     - DD_APM_NON_LOCAL_TRAFFIC=true # enables agent to receive traces from other containers
+     - DD_AGENT_HOST=dd-agent # allows web container to forward traces to agent
+     - DD_SITE=datadoghq.com # determines datadog instance to send data to (e.g change to datadoghq.eu for EU1)
+    volumes:
+     - /var/run/docker.sock:/var/run/docker.sock
+     - /proc/:/host/proc/:ro
+     - /sys/fs/cgroup:/host/sys/fs/cgroup:ro
+```
+
+The main changes to point above are the configuration of the `DD_AGENT_HOST` environment variable, which must be the same for our `web` container and our Agent container in order to collect traces. `DD_APM_ENABLED` and `DD_APM_NON_LOCAL_TRAFFIC` enable APM and allow our agent to receive traces from other containers. 
+
+Additionally we've added the `ddtrace` library to the `requirements.txt` for our Python web app. This is why we're able to initialize it with `ddtrace-run` to enable APM (the `datadog` library mentioned below is used for collection custom, DogStatsD metrics).
```

Comment:
```suggestion
This example also adds the `ddtrace` library to the `requirements.txt` for the Python web app so that you can initialize it with `ddtrace-run` to enable APM. (The `datadog` library mentioned in the following list is used to collect custom DogStatsD metrics.)
```